common/pkgfiles/referencedfilelist.cpp

```diff
@@ -378,7 +378,17 @@ void ReferencedFile::processForeignFileTree(IPropertyTree *tree, const char *src
     }
 }
 
-void ReferencedFile::processRemoteFileTree(IPropertyTree *tree, const char *srcCluster, StringArray *subfiles)
+StringBuffer &makeRemoteLFN(StringBuffer &remoteLfn, const char *remoteStorageName, const char *remotePrefix, const char *lfn)
+{
+    if (isEmptyString(remoteStorageName))
+        return remoteLfn;
+    remoteLfn.append("remote::").append(remoteStorageName).append("::");
+    if (remotePrefix && *remotePrefix)
```

Comment:
trivial: looks slightly odd that isEmptyString used 2 lines above, but not !isEmtpyString here.